osu.Framework/Graphics/Transforms/Transformable.cs

```diff
@@ -79,20 +79,29 @@ protected void UpdateTransforms()
 
         private double lastUpdateTransformsTime;
 
-        private readonly List<TargetMemberTransformTracker> targetMemberTrackers = new List<TargetMemberTransformTracker>();
+        private readonly List<TargetGroupingTransformTracker> targetGroupingTrackers = new List<TargetGroupingTransformTracker>();
 
-        private TargetMemberTransformTracker getTrackerFor(string targetMember, bool createIfNotExisting = false)
+        private TargetGroupingTransformTracker getTrackerFor(string targetMember)
         {
-            foreach (var t in targetMemberTrackers)
+            foreach (var t in targetGroupingTrackers)
             {
-                if (t.TargetMember == targetMember)
+                if (t.TargetMembers.Contains(targetMember))
                     return t;
             }
 
-            if (!createIfNotExisting) return null;
+            return null;
+        }
+
+        private TargetGroupingTransformTracker getTrackerForGrouping(string targetGrouping)
```

Comment:
This will get or create a tracker, contrary to `getTrackerFor()`, which can only get; would probably reflect that in method name.

```suggestion
        private TargetGroupingTransformTracker getOrCreateTrackerForGrouping(string targetGrouping)
```
(+ usages)

Reply:
yeah, was considering this but private method so didn't think much of it. sounds good.